index.js

```diff
@@ -1,14 +1,10 @@
 /**
- * Mnemonist Library Endpoint
- * ===========================
+ * Mnemonist Library Endpoint (CommonJS)
+ * ======================================
  *
  * Exporting every data structure through a unified endpoint. Consumers
  * of this library should prefer the modular access though.
  */
-var Heap = require('./heap.js'),
```

Comment:
Why did you remove those in favor of calling `require` multiple times? I know `require` etc. is usually cached and all. Is this to ease static evaluation or something of the kind?

Reply:
It can improve the handling tree-shaking but require to mix exports type